--- audit-3.0.8/init.d/auditd.service.orig	2022-05-21 09:13:09.864005082 +0200
+++ audit-3.0.8/init.d/auditd.service	2022-05-21 09:13:29.090567589 +0200
@@ -11,7 +11,6 @@ After=local-fs.target systemd-tmpfiles-s
 Before=sysinit.target shutdown.target
 ##Before=shutdown.target
 Conflicts=shutdown.target
-RefuseManualStop=yes
 ConditionKernelCommandLine=!audit=0
 ConditionKernelCommandLine=!audit=off
 
