--- audit-3.1.4/init.d/Makefile.am.orig	2024-05-19 09:59:22.005980091 +0200
+++ audit-3.1.4/init.d/Makefile.am	2024-05-19 10:14:22.091103912 +0200
@@ -29,12 +29,11 @@ EXTRA_DIST = auditd.init auditd.service
 	audit-stop.rules augenrules
 libconfig = libaudit.conf
 if ENABLE_SYSTEMD
-initdir = /usr/lib/systemd/system
+systemdunitdir = /lib/systemd/system
 legacydir = $(libexecdir)/initscripts/legacy-actions/auditd
-else
+endif
 initdir = $(sysconfdir)/rc.d/init.d
 sysconfigdir = $(sysconfdir)/sysconfig
-endif
 
 auditdir = $(sysconfdir)/audit
 auditrdir = $(auditdir)/rules.d
@@ -43,17 +42,14 @@ sbin_SCRIPTS = augenrules
 
 install-data-hook:
 	$(INSTALL_DATA) -D -m 640 ${srcdir}/${libconfig} ${DESTDIR}${sysconfdir}/${libconfig}
-if ENABLE_SYSTEMD
-else
 	$(INSTALL_DATA) -D -m 640 ${srcdir}/auditd.sysconfig ${DESTDIR}${sysconfigdir}/auditd
-endif
 
 install-exec-hook:
 if ENABLE_SYSTEMD
 	mkdir -p ${DESTDIR}${initdir}
 	mkdir -p ${DESTDIR}${legacydir}
 	mkdir -p ${DESTDIR}${libexecdir}
-	$(INSTALL_SCRIPT) -D -m 644 ${srcdir}/auditd.service ${DESTDIR}${initdir}/auditd.service
+	$(INSTALL_SCRIPT) -D -m 644 ${srcdir}/auditd.service ${DESTDIR}${systemdunitdir}/auditd.service
 	$(INSTALL_SCRIPT) -D -m 750 ${srcdir}/auditd.rotate ${DESTDIR}${legacydir}/rotate
 	$(INSTALL_SCRIPT) -D -m 750 ${srcdir}/auditd.resume ${DESTDIR}${legacydir}/resume
 	$(INSTALL_SCRIPT) -D -m 750 ${srcdir}/auditd.reload ${DESTDIR}${legacydir}/reload
@@ -61,16 +57,15 @@ if ENABLE_SYSTEMD
 	$(INSTALL_SCRIPT) -D -m 750 ${srcdir}/auditd.stop ${DESTDIR}${legacydir}/stop
 	$(INSTALL_SCRIPT) -D -m 750 ${srcdir}/auditd.restart ${DESTDIR}${legacydir}/restart
 	$(INSTALL_SCRIPT) -D -m 750 ${srcdir}/auditd.condrestart ${DESTDIR}${legacydir}/condrestart
-else
-	$(INSTALL_SCRIPT) -D ${srcdir}/auditd.init ${DESTDIR}${initdir}/auditd
 endif
+	$(INSTALL_SCRIPT) -D ${srcdir}/auditd.init ${DESTDIR}${initdir}/auditd
 	chmod 0755 $(DESTDIR)$(sbindir)/augenrules
 
 
 uninstall-hook:
 	rm ${DESTDIR}${sysconfdir}/${libconfig}
 if ENABLE_SYSTEMD
-	rm ${DESTDIR}${initdir}/auditd.service
+	rm ${DESTDIR}${systemdunitdir}/auditd.service
 	rm ${DESTDIR}${legacydir}/rotate
 	rm ${DESTDIR}${legacydir}/resume
 	rm ${DESTDIR}${legacydir}/reload
@@ -78,8 +73,7 @@ if ENABLE_SYSTEMD
 	rm ${DESTDIR}${legacydir}/stop
 	rm ${DESTDIR}${legacydir}/restart
 	rm ${DESTDIR}${legacydir}/condrestart
-else
+endif
 	rm ${DESTDIR}${sysconfigdir}/auditd
 	rm ${DESTDIR}${initdir}/auditd
-endif
 
