--- audit-userspace-4.1.2/init.d/auditd.service.in~	2025-09-02 03:57:57.000000000 +0200
+++ audit-userspace-4.1.2/init.d/auditd.service.in	2026-01-19 21:10:22.091850067 +0100
@@ -24,7 +24,6 @@ After=local-fs.target systemd-tmpfiles-s
 Before=sysinit.target shutdown.target audit-rules.service
 #Before=shutdown.target
 Conflicts=shutdown.target
-RefuseManualStop=yes
 
 Documentation=man:auditd(8) https://github.com/linux-audit/audit-documentation
 
