--- audit-3.1.1/configure.ac.orig	2023-04-27 19:26:56.000000000 +0200
+++ audit-3.1.1/configure.ac	2023-05-13 23:10:52.198788053 +0200
@@ -421,8 +421,6 @@ AC_ARG_WITH(libwrap,
 		AC_CHECK_HEADER(tcpd.h, [],
 			AC_MSG_ERROR([Could not find libwrap headers]),)
 		AC_CHECK_LIB(wrap, request_init, [ LIBWRAP_LIBS="-lwrap" ])
-		AC_CHECK_LIB(nsl, yp_get_default_domain, [
-			LIBWRAP_LIBS="$LIBWRAP_LIBS -lnsl" ])
 		;;
 	*)
 		AC_MSG_RESULT(yes)
@@ -434,8 +432,6 @@ AC_ARG_WITH(libwrap,
 		AC_CHECK_HEADER(tcpd.h, [],
 			AC_MSG_ERROR([Could not find libwrap headers]))
 		AC_CHECK_LIB(wrap, request_init, [])
-		AC_CHECK_LIB(nsl, yp_get_default_domain, [
-			LIBWRAP_LIBS="$LIBWRAP_LIBS -lnsl" ])
 		OLDLIBS="$LIBS"
 		LIBS="$LIBWRAP_LIBS $LIBS"
 		AC_LINK_IFELSE([AC_LANG_PROGRAM(
