--- audit-2.3.1/init.d/Makefile.am.orig	2013-06-08 22:26:09.755754441 +0200
+++ audit-2.3.1/init.d/Makefile.am	2013-06-08 22:28:05.405752030 +0200
@@ -28,12 +28,11 @@
 dispconfig = audispd.conf
 dispconfigdir = $(sysconfdir)/audisp
 if ENABLE_SYSTEMD
-initdir = /usr/lib/systemd/system
+systemdunitdir = /lib/systemd/system
 legacydir = $(libexecdir)/initscripts/legacy-actions/auditd
-else
+endif
 initdir = $(sysconfdir)/rc.d/init.d
 sysconfigdir = $(sysconfdir)/sysconfig
-endif
 
 auditdir = $(sysconfdir)/audit
 auditrdir = $(auditdir)/rules.d
@@ -44,22 +43,18 @@
 install-data-hook:
 	$(INSTALL_DATA) -D -m 640 ${srcdir}/${dispconfig} ${DESTDIR}${dispconfigdir}/${dispconfig}
 	$(INSTALL_DATA) -D -m 640 ${srcdir}/${libconfig} ${DESTDIR}${sysconfdir}/${libconfig}
-if ENABLE_SYSTEMD
-else
 	$(INSTALL_DATA) -D -m 640 ${srcdir}/auditd.sysconfig ${DESTDIR}${sysconfigdir}/auditd
-endif
 
 install-exec-hook:
 if ENABLE_SYSTEMD
 	mkdir -p ${DESTDIR}${initdir}
 	mkdir -p ${DESTDIR}${legacydir}
-	$(INSTALL_SCRIPT) -D -m 640 ${srcdir}/auditd.service ${DESTDIR}${initdir}/auditd.service
+	$(INSTALL_SCRIPT) -D -m 640 ${srcdir}/auditd.service ${DESTDIR}${systemdunitdir}/auditd.service
 	$(INSTALL_SCRIPT) -D -m 750 ${srcdir}/auditd.rotate ${DESTDIR}${legacydir}/rotate
 	$(INSTALL_SCRIPT) -D -m 750 ${srcdir}/auditd.resume ${DESTDIR}${legacydir}/resume
 	$(INSTALL_SCRIPT) -D -m 750 ${srcdir}/auditd.stop ${DESTDIR}${legacydir}/stop
-else
-	$(INSTALL_SCRIPT) -D ${srcdir}/auditd.init ${DESTDIR}${initdir}/auditd
 endif
+	$(INSTALL_SCRIPT) -D ${srcdir}/auditd.init ${DESTDIR}${initdir}/auditd
 	chmod 0750 $(DESTDIR)$(sbindir)/augenrules
 
 
@@ -67,9 +62,8 @@
 	rm ${DESTDIR}${dispconfigdir}/${dispconfig}
 	rm ${DESTDIR}${sysconfdir}/${libconfig}
 if ENABLE_SYSTEMD
-	rm ${DESTDIR}${initdir}/auditd.service
-else
+	rm ${DESTDIR}${systemdunitdir}/auditd.service
+endif
 	rm ${DESTDIR}${sysconfigdir}/auditd
 	rm ${DESTDIR}${initdir}/auditd
-endif
 
