--- audit-2.8.4/init.d/Makefile.am.orig	2018-08-29 16:26:20.634007070 +0200
+++ audit-2.8.4/init.d/Makefile.am	2018-08-29 17:08:25.793978367 +0200
@@ -30,12 +30,11 @@
 dispconfig = audispd.conf
 dispconfigdir = $(sysconfdir)/audisp
 if ENABLE_SYSTEMD
-initdir = /usr/lib/systemd/system
+systemdunitdir = /lib/systemd/system
 legacydir = $(libexecdir)/initscripts/legacy-actions/auditd
-else
+endif
 initdir = $(sysconfdir)/rc.d/init.d
 sysconfigdir = $(sysconfdir)/sysconfig
-endif
 
 auditdir = $(sysconfdir)/audit
 auditrdir = $(auditdir)/rules.d
@@ -45,16 +44,13 @@
 install-data-hook:
 	$(INSTALL_DATA) -D -m 640 ${srcdir}/${dispconfig} ${DESTDIR}${dispconfigdir}/${dispconfig}
 	$(INSTALL_DATA) -D -m 640 ${srcdir}/${libconfig} ${DESTDIR}${sysconfdir}/${libconfig}
-if ENABLE_SYSTEMD
-else
 	$(INSTALL_DATA) -D -m 640 ${srcdir}/auditd.sysconfig ${DESTDIR}${sysconfigdir}/auditd
-endif
 
 install-exec-hook:
 if ENABLE_SYSTEMD
-	mkdir -p ${DESTDIR}${initdir}
+	mkdir -p ${DESTDIR}${systemdunitdir}
 	mkdir -p ${DESTDIR}${legacydir}
-	$(INSTALL_SCRIPT) -D -m 644 ${srcdir}/auditd.service ${DESTDIR}${initdir}/auditd.service
+	$(INSTALL_SCRIPT) -D -m 644 ${srcdir}/auditd.service ${DESTDIR}${systemdunitdir}/auditd.service
 	$(INSTALL_SCRIPT) -D -m 750 ${srcdir}/auditd.rotate ${DESTDIR}${legacydir}/rotate
 	$(INSTALL_SCRIPT) -D -m 750 ${srcdir}/auditd.resume ${DESTDIR}${legacydir}/resume
 	$(INSTALL_SCRIPT) -D -m 750 ${srcdir}/auditd.reload ${DESTDIR}${legacydir}/reload
@@ -62,9 +58,8 @@
 	$(INSTALL_SCRIPT) -D -m 750 ${srcdir}/auditd.stop ${DESTDIR}${legacydir}/stop
 	$(INSTALL_SCRIPT) -D -m 750 ${srcdir}/auditd.restart ${DESTDIR}${legacydir}/restart
 	$(INSTALL_SCRIPT) -D -m 750 ${srcdir}/auditd.condrestart ${DESTDIR}${legacydir}/condrestart
-else
-	$(INSTALL_SCRIPT) -D ${srcdir}/auditd.init ${DESTDIR}${initdir}/auditd
 endif
+	$(INSTALL_SCRIPT) -D ${srcdir}/auditd.init ${DESTDIR}${initdir}/auditd
 	chmod 0750 $(DESTDIR)$(sbindir)/augenrules
 
 
@@ -72,7 +67,7 @@
 	rm ${DESTDIR}${dispconfigdir}/${dispconfig}
 	rm ${DESTDIR}${sysconfdir}/${libconfig}
 if ENABLE_SYSTEMD
-	rm ${DESTDIR}${initdir}/auditd.service
+	rm ${DESTDIR}${systemdunitdir}/auditd.service
 	rm ${DESTDIR}${legacydir}/rotate
 	rm ${DESTDIR}${legacydir}/resume
 	rm ${DESTDIR}${legacydir}/reload
@@ -80,8 +75,7 @@
 	rm ${DESTDIR}${legacydir}/stop
 	rm ${DESTDIR}${legacydir}/restart
 	rm ${DESTDIR}${legacydir}/condrestart
-else
+endif
 	rm ${DESTDIR}${sysconfigdir}/auditd
 	rm ${DESTDIR}${initdir}/auditd
-endif
 
