--- audit-1.2.6/init.d/Makefile.am.orig	2006-06-29 20:49:58.000000000 +0200
+++ audit-1.2.6/init.d/Makefile.am	2006-09-12 19:31:11.884451750 +0200
@@ -31,7 +31,7 @@
 
 install-data-hook:
 	$(INSTALL_DATA) -D -m 640 ${srcdir}/auditd.sysconfig ${DESTDIR}${sysconfigdir}/auditd
-	$(INSTALL_DATA) -D -m 640 ${srcdir}/${libconfig} ${DESTDIR}${sysconfdir}
+	$(INSTALL_DATA) -D -m 640 ${srcdir}/${libconfig} ${DESTDIR}${sysconfdir}/${libconfig}
 
 install-exec-hook:
 	$(INSTALL_SCRIPT) -D ${srcdir}/auditd.init ${DESTDIR}${initdir}/auditd
