--- audit-2.5.2/bindings/golang/Makefile.am.orig	2016-05-10 20:17:50.423996653 +0200
+++ audit-2.5.2/bindings/golang/Makefile.am	2016-05-10 20:27:31.460638934 +0200
@@ -25,15 +25,14 @@
 dist_check_SCRIPTS = test.go
 
 if HAVE_GOLANG
-LIBDIR = lib
-GODIR = $(LIBDIR)/golang/src/pkg/redhat.com/audit
+GODIR = $(libdir)/golang/src/redhat.com/audit
 
 install:
-	[ -d $(DESTDIR)${prefix}/$(GODIR) ] || mkdir -p $(DESTDIR)${prefix}/$(GODIR)
-	install -m 644 ${top_srcdir}/bindings/golang/audit.go $(DESTDIR)${prefix}/$(GODIR)
+	[ -d $(DESTDIR)$(GODIR) ] || mkdir -p $(DESTDIR)$(GODIR)
+	install -m 644 ${top_srcdir}/bindings/golang/audit.go $(DESTDIR)$(GODIR)
 
 uninstall:
-	@rm -f $(DESTDIR)${prefix}/$(GODIR)/*
+	@rm -f $(DESTDIR)$(GODIR)/*
 
 check:
 	@mkdir audit
